contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -641,6 +629,31 @@ export const Channel = new Resource({
       return response.data;
     });
   },
+  /**
+   * Ensure we merge content defaults when calling `update`
+   *
+   * @param {String} id
+   * @param {Object} [content_defaults]
+   * @param {Object} changes
+   * @return {Promise}
+   */
+  update(id, { content_defaults = {}, ...changes }) {
```

Comment:
Ahhhh, of course. We could also do this by specifying key paths for nested changes, rather than trying to write the whole object.

Reply:
Oooh, and then Dexie would interpret that?

Reply:
Yeah, can specify either nested objects or `.` separated keyPaths.